### PR TITLE
check setup created

### PR DESCRIPTION
**Pull request by jan-janssen**

### CODE REVIEW

File: pyiron/base/job/generic.py [hunk 1]
```diff
@@ -997,6 +1005,8 @@ def _run_if_new(self, debug=False):
             debug (bool): Debug Mode
         """
         self.validate_ready_to_run()
+        if self.server.run_mode.queue:
```
Comment (jan-janssen) [inline pyiron/base/job/generic.py]: Why do you only run the test if `run_mode` queue is selected?

Comment (samwaseda) [reply pyiron/base/job/generic.py]: The tests should be performed only when the job is submitted right?

Comment (jan-janssen) [reply pyiron/base/job/generic.py]: To have a consistent experience for the user I would recommend that we always run the check.

Comment (samwaseda) [reply pyiron/base/job/generic.py]: We decided together that it should be run only when the job is submitted

**Pull request merged**